### PR TITLE
Bug 1209041 - <gaia-header> update 

### PR DESCRIPTION
**Pull request by fabi1cazenave**
Deploy `<gaia-header>` version 0.9.1 in shared/elements and add a `dir="ltr"` attribute to all existing `<gaia-header>` occurrences in Gaia, except:
- apps/camera, because it uses its own (older) version of gaia-header
- apps/music, because it uses its own (older) version of gaia-header
- apps/system, because it has already been refactored and supports RTL navigation

https://bugzilla.mozilla.org/show_bug.cgi?id=1209041

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a290e2384b511f5f6fe45d0ca6d80d1f1cf6926d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3ba689cf5aac64fd55c9317a0603e4aab23dfe5c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d515cc1fef1ef93934b419e0f50330dc88f16f64

File: shared/elements/gaia-header/dist/gaia-header.js [hunk 1]
```diff
@@ -613,11 +693,11 @@ const MINIMUM_FONT_SIZE_CENTERED = 20;
  * This is the minimum font size that we can take
  * when the header title is not centered in the window.
  */
-const MINIMUM_FONT_SIZE_UNCENTERED = 18;
+const MINIMUM_FONT_SIZE_UNCENTERED = 16;
```
Comment (wilsonpage) [inline shared/elements/gaia-header/dist/gaia-header.js]: Hmm confused why this change has snuck in.

Comment (wilsonpage) [inline shared/elements/gaia-header/dist/gaia-header.js]: @julienw can you remeber why this change isn't on master yet?

Comment (wilsonpage) [inline shared/elements/gaia-header/dist/gaia-header.js]: Seems master is currently running `v0.8.0`. This patch updates to `v0.9.2`.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c58ed533e489f009d988011fa72e31e7ef9d9008

File: tv_apps/smart-system/fxa/test/marionette/test-fxa-client/js/fxa_iac_client.js [hunk 2]
```diff
@@ -1 +0,0 @@
-../../../../../../../shared/js/fxa_iac_client.js
```
Comment (wilsonpage) [inline tv_apps/smart-system/fxa/test/marionette/test-fxa-client/js/fxa_iac_client.js]: @fabi1cazenave not sure what happened to this file

Comment (fabi1cazenave) [inline tv_apps/smart-system/fxa/test/marionette/test-fxa-client/js/fxa_iac_client.js]: strange, indeed.

Comment (fabi1cazenave) [inline tv_apps/smart-system/fxa/test/marionette/test-fxa-client/js/fxa_iac_client.js]: looks like a github glitch, doesn’t it?

Comment (wilsonpage) [inline tv_apps/smart-system/fxa/test/marionette/test-fxa-client/js/fxa_iac_client.js]: Does it not show up on the CLi diff?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=797b380ea206fbf5ab3a4b201be1ad64d13f2761

**Comment by fabi1cazenave** [comment]
Just rebased to trigger another treeherder run…

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=32520159182f30d09d4787d6dea221120bb2297e

File: tv_apps/smart-system/fxa/test/marionette/test-fxa-client/js/fxa_iac_client.js (hunk 2 above)
Comment (fabi1cazenave) [inline tv_apps/smart-system/fxa/test/marionette/test-fxa-client/js/fxa_iac_client.js]: It seemed to be a file permission change. I don’t kwow where it came from, but I’ve reverted it.

File: shared/elements/gaia-header/dist/gaia-header.js (hunk 1 above)
Comment (julienw) [inline shared/elements/gaia-header/dist/gaia-header.js]: ahah yeah https://github.com/mozilla-b2g/gaia/pull/30264 never landed :) we landed the change in the v2.2 branch only.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7da8b26fda73938e39e0d7c1e43cdf646614aa55

**Comment by fabi1cazenave** [comment]
Quick rebase as the patch had bitrotten…

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=39589340f0cfd9a308c0e1148d2edbc5473686d8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f7e19f92a8624790094c6c611e18ccfe08663bd3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1b0058f9555067e964842e955971a6a1248b36d2

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=df596790438dea2745d1357c204e6346009a770c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4f25dec7c158dde834fc4bfa3cbcb0d49fd1652c

**Comment by fabi1cazenave** [comment]
Just bower-updated the components and rebased.

@wilsonpage please have a look before merging. :-)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=645fad6f9080651a1b79a330ebb0695709b68755

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=88e72be10cc4e47e9ca342abfebdc17c6b49bf9b
